src/pages/home/Home.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Heading from '../../atoms/Heading'
+import Grid from '../../atoms/Grid'
+import ArtistList from './components/ArtistList'
+
+const artistLink = [
+  {
+    name: 'Linkin Park',
+    id:
```

Comment:
Please remove the whole URL, and keep only artist's id in as the value:

```diff
- id: 'https://spotify-proxy-57097.herokuapp.com/v1/artists/4dwdTW1Lfiq0cM8nBAqIIz',
+ id: '4dwdTW1Lfiq0cM8nBAqIIz',
```

Reply:
You don't have to include a backend URL here, you won't be talking with backend. You just direct to the arist page on _your website_. That page already communicates with the backend to get the artist info.